components/loading/LoadingBar.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-app ref="loaderContainer" class="v-app">
+    <v-progress-linear
+      :active="loading"
+      :indeterminate="indeterminate"
+      :height="4"
+      :bottom="true"
+      :rounded="true"
+      :color="barColor"
+      :value="barValue"
+    ></v-progress-linear>
+  </v-app>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+let timeout: number;
```

Comment:
Looking at it again, I think this should be a `data`.

Being manipulated externally or in other components has no bearing on where it will live, imo. If it's part of what the component needs to work, it should be inside of the components.